extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -69,15 +34,15 @@ public function extendImageData( $data, $width ) {
 			$width, self::MAX_DESKTOP_THUMBNAIL_HEIGHT, $originalWidth, $file->getHeight() );
 		// if custom and big enough, scale thumbnail size
 		$ratio = !empty( $wgPortableInfoboxCustomImageWidth ) && $originalWidth > $wgPortableInfoboxCustomImageWidth ?
-			$wgPortableInfoboxCustomImageWidth / $dimensions[ 'width' ] : 1;
+			$wgPortableInfoboxCustomImageWidth / $dimensions['width'] : 1;
 		// get thumbnail
 		$thumbnail = $file->transform( [
-			'width' => round( $dimensions[ 'width' ] * $ratio ),
-			'height' => round( $dimensions[ 'height' ] * $ratio )
+			'width' => round( $dimensions['width'] * $ratio ),
+			'height' => round( $dimensions['height'] * $ratio )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "height" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)